tools/clone_flutter.sh

```diff
@@ -16,8 +16,17 @@ fi
 # Go to the engine git repo to get the date of the latest commit.
 cd $ENGINE_PATH/src/flutter
 
-# Special handling of release branches.
-ENGINE_BRANCH_NAME=$CIRRUS_BASE_BRANCH
+# Special handling of release branches. We would like to run the tests against
+# the release branch of flutter.
+#
+# On presubmit, we can get the branch name from the `CIRRUS_BASE_BRANCH` flag.
+# During the commit tests the base branch value is empty, instead
+# `CIRRUS_BRANCH` has the correct branch name.
+ENGINE_BRANCH_NAME="$CIRRUS_BASE_BRANCH"
+if [[ -z "$CIRRUS_BASE_BRANCH" ]] then
```

Comment:
is this the only way to determine we're running post submit?

Reply:
I only saw two flags in Cirrus. @christopherfujino do you know any other way of determining it?

Reply:
this is the only way I know of.

Reply:
not sure his github handle, so i pinged Fedor in the discord chat

Reply:
```suggestion
if [[ -z "$CIRRUS_BASE_BRANCH" ]]; then
```

Reply:
done. thanks!